### PR TITLE
The Witness: Option Groups & Tooltip formatting

### PR DESCRIPTION
**Pull request by NewSoupVi**
uwu *notices your options*

### CODE REVIEW

**Comment by NewSoupVi** [comment]
What did I do lmao

**Comment by NewSoupVi** [comment]
I can't have "single linebreaks" anymore because the template yaml breaks???

No. https://github.com/ArchipelagoMW/Archipelago/pull/3343

**Review by ScipioWright — APPROVED** [review]
Looks fine on webhost, no issues.

File: worlds/witness/options.py [hunk 1]
```diff
@@ -176,7 +178,7 @@ class VictoryCondition(Choice):
 
 class PuzzleRandomization(Choice):
     """
-    Puzzles in this randomizer are randomly generated. This option changes the difficulty/types of puzzles.
+Puzzles in this randomizer are randomly generated. This option changes the difficulty/types of puzzles.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:181]:
```suggestion
    Puzzles in this randomizer are randomly generated. This option changes the difficulty/types of puzzles.

```

**Review by Exempt-Medic — APPROVED** [review]
Changes LGTM and display well when running WebHost

**Review by LegendaryLinux — APPROVED** [review]
Even though I'm checking out and verifying all these changes anyway, I would appreciate screenshots next time. The template does ask for them, and this did make graphical changes.

**Pull request merged**